src/Maker/MakeResetPassword.php

```diff
@@ -0,0 +1,277 @@
+<?php
+
+/*
+ * This file is part of the Symfony MakerBundle package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\MakerBundle\Maker;
+
+use Symfony\Bundle\MakerBundle\ConsoleStyle;
+use Symfony\Bundle\MakerBundle\DependencyBuilder;
+use Symfony\Bundle\MakerBundle\Exception\RuntimeCommandException;
+use Symfony\Bundle\MakerBundle\FileManager;
+use Symfony\Bundle\MakerBundle\Generator;
+use Symfony\Bundle\MakerBundle\InputConfiguration;
+use Symfony\Bundle\MakerBundle\Security\InteractiveSecurityHelper;
+use Symfony\Bundle\MakerBundle\Util\YamlSourceManipulator;
+use Symfony\Bundle\MakerBundle\Validator;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use SymfonyCasts\Bundle\ResetPassword\SymfonyCastsResetPasswordBundle;
+
+class MakeResetPassword extends AbstractMaker
+{
+    /**
+     * @var FileManager
+     */
+    private $fileManager;
+
+    public function __construct(FileManager $fileManager)
+    {
+        $this->fileManager = $fileManager;
+    }
+
+    public static function getCommandName(): string
+    {
+        return 'make:reset-password';
+    }
+
+    public function configureCommand(
+        Command $command,
+        InputConfiguration $inputConfig
+    ) {
+        $command
+            ->setDescription('Create controller, entity, and repositories for use with SymfonyCasts Reset Password Bundle.')
+        ;
+    }
+
+    public function configureDependencies(DependencyBuilder $dependencies)
+    {
+        $dependencies->addClassDependency(SymfonyCastsResetPasswordBundle::class, 'symfonycasts/reset-password-bundle');
+    }
+
+    public function interact(InputInterface $input, ConsoleStyle $io, Command $command)
+    {
+        $io->title('Reset Password Bundle Requires:');
+        $requirements[] = '1) A user entity has been created.';
+        $requirements[] = '2) The user entity contains an email property with a getter method.';
+        $requirements[] = '3) A user repository exists for the user entity.'."\n";
+        $requirements[] = '<fg=yellow>bin/console make:user</> will generate the user entity and it\'s repository...'."\n";
+        $io->text($requirements);
```

Comment:
You're doing some nice work with the formatting!

But I'm not sure we need this entire section. What if we:

A) Move the "user" questions to be the *first* questions
B) Removed this section

Then the user should be able to naturally discover that they're missing the `User` class (or are missing an email field somehow) very early.

Reply:
done